aws/resource_aws_organization.go

```diff
@@ -0,0 +1,109 @@
+package aws
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/organizations"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	"log"
+)
+
+func resourceAwsOrganization() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsOrganizationCreate,
+		Read:   resourceAwsOrganizationRead,
+		Update: resourceAwsOrganizationUpdate,
```

Comment:
As written, this resource does not support updates. We should remove this line from the resource and its associated function.